src/database/DataTypes/TransactionBatch.js

```diff
@@ -10,6 +11,10 @@ export class TransactionBatch extends Realm.Object {
     return this.numberOfPacks * this.packSize;
   }
 
+  get itemCode() {
```

Comment:
Used in `ExternalSupplierInvoice.js`, since table rows are transactionBatches and not transactionItems like most other pages.. and no reference to item.code from TransactionBatch realm schema..have to improvise (itemCode should really be added to TransactionBatch and db migrated.. should I ?)